staff/carlos-corredor/blog-semantics/entries.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```

Comment:
@Carlos7979 folder name should be blog instead of blog-semantics, please update asap and inform us when it's done. Thanks!

Reply:
Understood!